extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -57,6 +60,46 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return this._initializeDeferred.promise;
 	}
 
+	get _visitedNotebooks(): string[] {
+		return this._extensionContext.globalState.get(constants.visitedNotebooksMementoKey, []);
+	}
+
+	set _visitedNotebooks(value: string[]) {
+		this._extensionContext.globalState.update(constants.visitedNotebooksMementoKey, value);
+	}
+
+	trustBook(bookTreeItem?: BookTreeItem): void {
+		let bookPathToTrust = bookTreeItem ? bookTreeItem.root : this.currentBook?.bookPath;
+
+		if (bookPathToTrust) {
+			let trustChanged = this._bookTrustManager.setBookAsTrusted(bookPathToTrust);
+
+			if (this._apiWrapper) {
+				if (trustChanged) {
+					this._apiWrapper.showInfoMessage(loc.msgBookTrusted);
+				} else {
+					this._apiWrapper.showInfoMessage(loc.msgBookAlreadyTrusted);
+				}
+			}
+		}
+	}
+
+	untrustBook(bookTreeItem?: BookTreeItem): void {
```

Comment:
There's still some untrusting code here

Reply:
Got it. Had left it there intentionally as an API provision, but yes, not really used. Will remove this from here, but will keep it in the trustmanager but left a method in the trust manager since that one is used by the tests.